external/vcm/vcm/calc/_zenith_angle.py

```diff
@@ -122,10 +118,7 @@ def _greenwich_mean_sidereal_time(model_time):
 
     theta_radians = np.deg2rad(theta / 240.0) % (2 * np.pi)
 
-    if theta_radians < 0:
```

Comment:
I actually don't think it is possible for `theta_radians` to be less than zero at this point since it is constructed using the modulo operator (a.k.a. the `np.remainder` function).  According to [the NumPy documentation](https://numpy.org/doc/stable/reference/generated/numpy.remainder.html):

> Computes the remainder complementary to the floor_divide function. It is equivalent to the Python modulus operator ``x1 % x2`` and has the same sign as the divisor x2.

In this case the divisor is always `2 * np.pi`, which is greater than zero, so the result should always be greater than or equal to zero.  I don't think there is much harm in keeping this check in the form I translated it to below, however, if we'd rather not change the algorithm.  I assume this check was originally here for a reason (though maybe it was that this code was ported from another language that had different behavior?).

Reply:
Yes, the modulo operator above should be equivalent to these two lines. For the same reason, is `+ 2 * np.pi * (theta_radians < 0)` necessary?

Reply:
Good, just wanted to make sure that my understanding was correct there.  I'll remove the `+ 2 * np.pi * (theta_radians < 0)` bit since it's a no-op.